lib/modules/manager/mix/extract.ts

```diff
@@ -28,26 +30,27 @@ export async function extractPackageFile(
       } while (!contentArr[lineNumber].includes('end'));
       let depMatchGroups = depMatchRegExp.exec(depBuffer)?.groups;
       while (depMatchGroups) {
-        const { depName, datasource, currentValue, organization } =
-          depMatchGroups;
+        const { app, requirement, opts } = depMatchGroups;
+
+        const github = githubRegexp.exec(opts)?.groups?.value;
+        const organization = organizationRegexp.exec(opts)?.groups?.value;
+
+        const depName = app;
```

Comment:
can we use the old name inside regex to not need this ?

Reply:
I wanted the regex to reflect [how `mix deps` defines the dependency tuple](https://hexdocs.pm/mix/Mix.Tasks.Deps.html), and only later map it to the `PackageDependency` structure.

I can do:

```typescript
const dep: PackageDependency = {
  depName: app,
  currentValue,
  datasource,
};
```

below if you think that's better.